.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
Confirmed, you don't need this as a separate file: https://babeljs.io/docs/usage/babelrc/ (see below for alternative)